modules/geo-layers/package.json

```diff
@@ -30,7 +30,10 @@
     "prepublishOnly": "npm run build-bundle && npm run build-bundle -- --env.dev"
   },
   "dependencies": {
-    "@loaders.gl/3d-tiles": "^2.0.2",
+    "@loaders.gl/core": "^2.1.0-alpha.4",
+    "@loaders.gl/3d-tiles": "^2.1.0-alpha.4",
+    "@loaders.gl/images": "^2.1.0-alpha.4",
```

Comment:
- Move `@loaders.gl/core` to `peerDependencies` (duplicate copies will cause problems)
- Is `@loaders.gl/images` directly referenced?

Reply:
`@loaders.gl/images` doesn't seem directly referenced. Should I remove it?

Reply:
Yes please.